doc-content/enterprise-only/websphere/was-data-source-create-proc.adoc

```diff
@@ -3,6 +3,10 @@
 
 A data source is an object that enables a Java Database Connectivity (JDBC) client, such as an application server, to establish a connection with a database. Applications look up the data source on the Java Naming and Directory Interface (JNDI) tree or in the local application context and request a database connection to retrieve data. You must configure data sources for {WEBSPHERE} to ensure proper data exchange between the servers and the designated database.
 
+Solutions using {PRODUCT} integration manage different resources within a single transaction. JMS for async jobs, events, and timers, for example. {PRODUCT} requires an XA driver in the datasource when possible to ensure data atomicity and predictable results. If transactional code for different schemas exists inside listeners or derives from hooks provided by the jBPM engine, an XA driver is also required. 
```

Comment:
[duplication - and rephrase suggestion]

If possible, see if there is a way to reduce the duplication. As these are opening paragraph sections, perhaps they can be parted out as a concept .adoc?